lib/tasks/occupation.rake

```diff
@@ -10,27 +10,28 @@ namespace :occupation do
         next if index.zero?
         occupation = Occupation.find_or_initialize_by(name: row["RAPIDS TITLE"])
         onet_code = OnetCode.find_by(code: row["ONET SOC CODE"].strip)
-        hybrid_start_hours = nil
-        hybrid_end_hours = nil
-        hybrid_hours = nil
+        hybrid_hours_min = nil
+        hybrid_hours_max = nil
 
         if row["HYBRID"]
           hours = row["HYBRID"]
-          hybrid_start_hours = hours.match(/(\d{1,})\s*-/)
-          hybrid_end_hours = hours.match(/-\s*(\d{1,})/)
+          hybrid_hours = hours.match(/(\d{1,})\s*-\s*(\d{1,})/)
```

Comment:
**nit**: I'm a big fan of named captures, that allow us to assign a meaning name to every capture group in the regex

```ruby
hybrid_hours = /(?<hybrid_hours_min>\d{1,})\s*-\s*(?<hybrid_hours_max>\d{1,})/
if hybrid_hours
  hybrid_hours_min = hybrid_hours[:hybrid_hours_min]
  hybrid_hours_max = hybrid_hours[:hybrid_hours_max]
else
...
```

Reply:
Oh cool, I didn't even know about this! I will update.